tools/metrics/prometheus/prometheus.yml

```diff
@@ -3,6 +3,6 @@ global:
   evaluation_interval: 1s
 
 scrape_configs:
-  - job_name: buildbuddy_server
+  - job_name: buildbuddy-app
```

Comment:
FYI this change is to ensure the local job matches our dev jobs, which simplifies the dashboard query.